datacontract/imports/glue_importer.py

```diff
@@ -253,30 +267,20 @@ def map_type_from_sql(sql_type: str) -> str:
         return None
 
     sql_type = sql_type.lower()
-    if sql_type.startswith("varchar"):
-        return "varchar"
-    if sql_type.startswith("string"):
-        return "string"
-    if sql_type.startswith("text"):
-        return "text"
-    if sql_type.startswith("byte"):
-        return "byte"
-    if sql_type.startswith("short"):
-        return "short"
-    if sql_type.startswith("integer") or sql_type.startswith("int"):
-        return "integer"
-    if sql_type.startswith("long") or sql_type.startswith("bigint"):
-        return "long"
-    if sql_type.startswith("float"):
-        return "float"
-    if sql_type.startswith("double"):
-        return "double"
-    if sql_type.startswith("boolean"):
-        return "boolean"
-    if sql_type.startswith("timestamp"):
-        return "timestamp"
-    if sql_type.startswith("date"):
-        return "date"
-    if sql_type.startswith("decimal"):
-        return "decimal"
-    return "variant"
+
+    type_mapping = {
+        "string": "string",
+        "int": "int",
+        "bigint": "bigint",
```

Comment:
Issue https://github.com/datacontract/datacontract-cli/issues/346 with `bigint` -> `long` conversion has been fixed. Also, some data types that are not used by Glue have been removed - `short`, `byte`, `text`. 
Also, `integer` has been renamed to `int` to be consistent with Glue (not backward compatible change).
Notice that `variant` was renamed to `unknown`, this is what Glue uses when something went wrong
![image](https://github.com/user-attachments/assets/05f6f99f-85bf-447d-8e84-bce2bc59d69c)